packages/lib-classifier/src/components/Classifier/components/Feedback/components/feedback-modal.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Translate from 'react-translate-component';
+import counterpart from 'counterpart';
+import FrameViewer from '../../../../components/frame-viewer';
+import ModalFocus from '../../../../components/modal-focus';
+
```

Comment:
Most of these imports aren't used in the new classifier, so we're better off just building a new Feedback component from scratch.

Reply:
I'm inclined to just delete 2d81f7177117d7942ccbe7a22d98365fb19791a7. I don't think we're going to use any of the files in that commit.